### ISSUE TITLE
Improve feedback when empty results are returned from GetFeatureInfo Requests

### ISSUE THREAD

**Issue by MV88** [geosolutions-it/MapStore2#7276]
## Description

Actually an empty feature info viewer is displayed when you click on this map that has 2  
layers that points to geocollect geoserver instance
Due to a proxy property missing (see [here](https://github.com/geosolutions-it/MapStore2-C040/issues/445#issuecomment-904527487)) they return an empty body

what we can do is to show some message for this use case

so open [this](https://dev-mapstore.geosolutionsgroup.com/mapstore/#/viewer/openlayers/35736) map and click on a feature

expected: 
to see the message notifying empty results has been returned from the server

current 
empty white page

**What kind of improvement you want to add?** (check one with "x", remove the others)
- [ ] Minor changes to existing features
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**Comment by MV88** [comment]
The requirements for this feature have changed:

Basically we should avoid to manage and show dedicated messages for empty responses
instead we will treat them as failing responses  @offtherailz  please confirm this

so if we do that we should obtain:
![image](https://user-images.githubusercontent.com/11991428/132889372-c7e0714d-8860-4c86-a646-da64bd8e620f.png)

instead of current behavior

![image](https://user-images.githubusercontent.com/11991428/132889400-d3555b54-de05-4ceb-849d-f39e058b0199.png)

**Comment by MV88** [comment]
@tdipisa we can postpone this issue and since it depends on #7296
